packages/core/strapi/lib/Strapi.js

```diff
@@ -78,7 +103,7 @@ class Strapi {
 
       return this;
     } catch (error) {
-      return this.stopWithError(error.message);
+      return this.stopWithError(error);
```

Comment:
Does this really work ?

Reply:
Worked for me. Here is the code of `stopWithError`;
```js
  stopWithError(err, customMessage) {
    this.log.debug(`⛔️ Server wasn't able to start properly.`);
    if (customMessage) {
      this.log.error(customMessage);
    }

    this.log.error(err);
    return this.stop();
  }
```

Why wouldn't it work?